Monika After Story/game/event-handler.rpy

```diff
@@ -2655,19 +2655,12 @@ label prompt_menu:
         #   restored on a relaunch, which would make no sense lol.
 
         # only call label if it exists
-        if cb_label is not None:
+        if renpy.has_label(cb_label):
             call expression cb_label
```

Comment:
idle callbacks might want access to their data (`_mas_idle_data` - see the other comment for more info), so tearing it down before the label is called is not going to work here.